### PR TITLE
Updated with my contribution

### PR DESCRIPTION
**Pull request by TrongHieu90**
Updated with my contribution of this challenge in WF c#

### CODE REVIEW

File: _CodingChallenges/051.1-astar.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ contributions:
     source: "https://github.com/MkLahane/Game-Development"
     author:
       name: "Makarand Lahane"
+  - title: "Pathfinding A* in Windows Form using C#"
+    author:
+      name: "Trong Hieu Nguyen"
+      url:
+    url: "https://github.com/TrongHieu90/PathfindingAStar"
+    source: "https://github.com/TrongHieu90/PathfindingAStar"
+
```
Comment (nilsweb) [inline _CodingChallenges/051.1-astar.md]: Line `41` and `42` contain whitespace. Tests should pass as soon as they are removed!

Comment (TrongHieu90) [reply _CodingChallenges/051.1-astar.md]: White Space has been removed in these 2 lines.

File: _CodingChallenges/051.1-astar.md [hunk 2]
```diff
@@ -32,6 +32,12 @@ contributions:
     source: "https://github.com/MkLahane/Game-Development"
     author:
       name: "Makarand Lahane"
+  - title: "Pathfinding A* in Windows Form using C#"
+    author:
+      name: "Trong Hieu Nguyen"
+      url:
```
Comment (nilsweb) [inline _CodingChallenges/051.1-astar.md]: Sorry, but I totally overlooked this. If you don't want to specify a URL here, just leave this line out.

**Comment by gruselhaus** [comment]
@TrongHieu90 
Hi, 
I solved the conflicts. Can you take a look at this and a go to merge?

**Comment by nilsweb** [comment]
Thank you! :+1:

**Pull request merged**